desc/coils.py

```diff
@@ -148,7 +148,7 @@ def current(self, new):
         self._current = float(np.squeeze(new))
 
     def compute_magnetic_field(
-        self, coords, params=None, basis="rpz", source_grid=None
+        self, coords, params=None, basis="rpz", source_grid=None, transforms=None
```

Comment:
what's actually needed in transforms to compute the magnetic field?

Reply:
If the field is `CurrentPotentialField`  or `FourierCurrentPotentialField`, then it needs geometry info of the surface so R and Z transforms are needed, and if it is FourierCurrentPotentialField it also needs Phi transform (bc Phi is itself a DoubleFourierSeries